src/qpython.cpp

```diff
@@ -127,6 +129,103 @@ QPython::addImportPath(QString path)
 }
 
 void
+QPython::importNames(QString name, QVariant args, QJSValue callback)
+{
+    QJSValue *cb = 0;
+    if (!callback.isNull() && !callback.isUndefined() && callback.isCallable()) {
+        cb = new QJSValue(callback);
+    }
+    emit import_names(name, args, cb);
+}
+
+bool
+QPython::importNames_sync(QString module_name, QVariant args)
+{
+    // The plan is:
+    // import the module, given by module_name,
+    // don't put it into globals of priv,
+    // make some temporary dict with globals + the new module,
+    // get the callables from the temp globals, given by names in args,
+    // put the callables into globals of priv
+    // call it a day
+    // (then one could reimplement importModule based on importNames)
+
+    // getting the module (the code is from importModule_sync)
+    QByteArray utf8bytes = module_name.toUtf8();
+    const char *moduleName = utf8bytes.constData();
+
+    ENSURE_GIL_STATE;
+
+    bool use_api_10 = (api_version_major == 1 && api_version_minor == 0);
```

Comment:
This is new API so it didn't exist with 1.0, right? No need to differentiate between APIs.